lib/executor/ramping_vus.go

```diff
@@ -49,18 +49,11 @@ func init() {
 	)
 }
 
-// Stage contains
-type Stage struct {
-	Duration types.NullDuration `json:"duration"`
-	Target   null.Int           `json:"target"` // TODO: maybe rename this to endVUs? something else?
-	// TODO: add a progression function?
```

Comment:
Not sure why you decided to remove this - as this `TODO` says, we have a custom `stage` here, since we plan to add the possibility for non-linear progression functions (which won't be available in the simple `options.stages` configs), and maybe even things like custom tags. Also, `lib.Stage` has some extra cruft (see `lib.StageFields`) related to the CLI flags parsing that we simply don't need here...

It's not a big deal, we can always refactor it back and split it again, I just don't see the problem with such a simple type being defined locally in the executors.

Reply:
I needed it to move `SumStagesDuration` to `lib`, and removing this duplication made sense, but I see your point, and I'm no longer calling `SumStagesDuration` from `cmd`, so I reverted this in cc01c16e.